www/js/controller.js

```diff
@@ -1081,19 +1081,18 @@ define([
 		*/
 		logSolutionStep: function(obj){
 			//Stub for logging the messages, updated in con-author and con-student
+			if(!obj.property) return null;
+
 			var property = obj.property;
 			// if node ID was not in the object then send node ID as the current ID.
 			// used for auto created nodes.
 			if(!obj.nodeID){
 				obj.nodeID = this.currentID;
 				obj.node = this._model.authored.getVariable(this.currentID);
 			}
-			if(property){
-				if(!obj.value) obj.value = registry.byId(this.controlMap[property]).get("value");
-				obj.type = this._model.active == this._model.authored ? "solution-enter" : "solution-check";
-			} else {
-				return null;
-			}
+			if(!obj.value) obj.value = registry.byId(this.controlMap[property]).get("value");
+			obj.type = this._model.active == this._model.authored ? "solution-enter" : "solution-check";i
```

Comment:
There's an extra "i" at the end of this line...

When we get our build process going, we can make sure you all can run it to catch typos like this automatically. 😄